Sources/MongoSwift/BSON/BSONDocument.swift

```diff
@@ -255,7 +255,7 @@ extension BSONDocument {
             throw BSONError.InternalError(message: "No keys to exclude, use 'bson_copy' instead")
         }
 
-        let cStrings: [ContiguousArray<CChar>] = keys.map { $0.utf8CString }
+        let cStrings: [ContiguousArray<CChar>] = keys.map(\.utf8CString)
```

Comment:
new SwiftFormat rule that prefers using keypaths where possible -- I like it so I left it on for now, LMK what you think